test/Deck-test.js

```diff
@@ -0,0 +1,57 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Deck = require('../src/Deck');
+const Card = require('../src/Card');
+
+describe('Deck', function() {
+  it('should be a function', function() {
+    expect(Deck).to.be.a('function');
+  });
+
+  it('should be an instance of Deck', function() {
+    const deck = new Deck();
+    expect(deck).to.be.an.instanceof(Deck);
+  });
+
+  it('should store Cards in array', function() {
+    const card1 = new Card(
+      1,
+      'What is Robbie\'s favorite animal',
+      ['sea otter', 'pug', 'capybara'],
+      'sea otter');
+    const card2 = new Card(
+      14,
+      'What organ is Khalid missing?',
+      ['spleen', 'appendix', 'gallbladder'],
+      'gallbladder');
+    const card3 = new Card(
+      12,
+      'What is Travis\'s middle name?',
+      ['Lex', 'William', 'Fitzgerald'],
+      'Fitzgerald');
+    const deck = new Deck([card1, card2, card3]);
+    expect(deck.cards).to.deep.equal([card1, card2, card3]);
```

Comment:
I am not sure if I should be writing this test differently isnce it seems "too easy", but I also may just be getting comfortable with writing tests and code for simple class constructors.